tests/test-pow.c

```diff
@@ -112,6 +112,12 @@ int main()
         doThePoW(PoW_Context_ptr, pow_ctx);
         int8_t *ret_trytes = getPoWResult(PoW_Context_ptr, pow_ctx);
         assert(ret_trytes);
+#if defined(ENABLE_STAT)
```

Comment:
I think the statistic should be printed after checking the PoW result is correct.

Reply:
Due to the PoW information would be freed or destroyed after `destroyImplContext` function, I print the statistics in this order.

I'll declare variables to store the PoW information and print them after the verification.